docs/tasks/federation/set-up-cluster-federation-kubefed.md

```diff
@@ -24,52 +24,57 @@ This guide assumes that you have a running Kubernetes cluster. Please
 see one of the [getting started](/docs/setup/) guides
 for installation instructions for your platform.
 
-
 ## Getting `kubefed`
 
-Download the client tarball corresponding to the latest release and
-extract the binaries in the tarball with the commands:
+Download the client tarball corresponding to the particular release and 
+extract the binaries in the tarball:
 
-```shell
-# Linux
-curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/kubernetes-client-linux-amd64.tar.gz
-tar -xzvf kubernetes-client-linux-amd64.tar.gz
+> Note that until kubernetes versions `1.8.x` the federation project was 
+maintained as part of [core kubernetes repo](https://github.com/kubernetes/kubernetes).
+At some point between kuberntes releases `1.8.0` and `1.9.0`, it moved into 
```

Comment:
kuberntes typo

Reply:
thanks @tengqm for checking this. Updated!